about-page.html

```diff
@@ -0,0 +1,236 @@
+            </h5>
+            <a class="contact-email">e3gaming2022@gmail.org</a>
+          </div>
+        </div>
+        <div class="about-logo-container dflex">
+          <h2 class="about-section-title">
+            Electronic Gaming Global Event 2022 Logo
+          </h2>
+          <hr class="redline-hr" />
+          <p class="logo-description">
+            The logo of E3 Gaming Event 2022 was decided through the logo of the
+            Los Angeles E3 Event.
+          </p>
+          <div class="logo-img-container dflex">
+            <img src="img/e3-logo.png" alt="E3 Company logo" />
+          </div>
+        </div>
+        <div id="past-events" class="about-past-container dflex">
+          <h2 class="about-section-title">See the past E3 Gaming Events!</h2>
+          <hr class="redline-hr" />
+          <p class="past-description">
+            Take a look at the last two E3 Gaming Events which took place in
+            Buenos Aires and in Los Angeles.
+          </p>
+          <ul class="event-list dflex">
+            <li class="dflex">
+              <div class="event-container dflex">
+                <img src="img/e3-event-2019.png" alt="E3 Gaming Event 2019" />
+                <h3 class="event-title centered">
+                  2019 <br />E3 Gaming Event in Buenos Aires
+                </h3>
+              </div>
+            </li>
+            <li class="dflex">
+              <div class="event-container dflex">
+                <img src="img/e3-event-2020.png" alt="E3 Gaming Event 2020" />
+                <h3 class="event-title centered">
+                  2020 <br />E3 Gaming Event in Los Angeles
+                </h3>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="about-footer-partner dflex">
+        <h2 class="section-title">Partner</h2>
+        <hr class="redline-hr" />
+        <ul class="partner-list dflex">
+          <li class="dflex">
+            <i class="fab fa-playstation"></i>
+          </li>
+          <li class="dflex">
+            <i class="fab fa-xbox"></i>
+          </li>
+          <li class="dflex">
+            <i class="fab fa-microsoft"></i>
+          </li>
+          <li class="dflex">
+            <i class="fab fa-steam"></i>
+          </li>
+          <li class="dflex">
+            <i class="fab fa-twitch"></i>
+          </li>
+        </ul>
+      </div>
+      <div class="about-company-rights dflex">
+        <div class="company-img-container dflex">
+          <img src="img/e3-logo.png" alt="E3 Company Logo" />
+          <p class="p-rights">
+            2015 Electronic Entertainment Expo. Some rights reserved.<br />License
+            and Credits to Cindy Shin:
+            <a
+              href="https://www.behance.net/adagio07"
+              >https://www.behance.net/adagio07</a
+            >
+          </p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Hi. Good job on the footer but there is a little thing that is missing. It is a requirement that on mobile view the background color of the footers for the 2 pages become white but on the mobile view of the home page, the footer is not even present. Please kindly fix that.

Reply:
Thank's for the advice, I will improve my code. 😄